contract/contracts/BoardOfDirectors.sol

```diff
@@ -189,6 +219,11 @@ contract BoardOfDirectors is ReentrancyGuardUpgradeable {
     require(success, 'Call failed');
 
     _action.isExecuted = true;
+    executedActions[_actionId] = _action;
+    delete pendingActions[_actionId];
```

Comment:
What does this do?

Reply:
when it's executed it adds to the executedActions and remove from pendingActions, but I need to test it first